### DISCUSSION TITLE
Add Linux-rt Kernel Option

### DISCUSSION THREAD

**Discussion by DrVektor** [archlinux/archinstall#2139] [Ideas]
There is no linux-rt (realtime kernel) option. Why? Maybe it can be. Thank you.

**Comment by Torxed** [comment]
We've simply missed that it moved out of AUR end of last year. We could add it in a PR :)

**Comment by codefiles** [comment]
There is also `linux-rt-lts`. I have been meaning to add them but have prioritized fixes over some of the stuff I want to add.
